.github/workflows/api.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           node-version: 'v20.12.0'
 
       - name: Install tsp
-        run: npm install -g @typespec/compiler@0.55.0
+        run: npm install -g @typespec/compiler@0.57.0
```

Comment:
should we just use `npm i` and then create an npm script in `package.json` to run tsp?

Reply:
Otherwise we're going to have to remember to tie `package.json` to whatever is in this file.